src/fft.jl

```diff
@@ -1,63 +1,121 @@
 import FFTW
 
+# returns the lengths of the bounding rectangle in reciprocal space
+# that encloses the sphere of radius Gmax
+function bounding_rectangle(lattice::AbstractMatrix{T}, Gmax; tol=1e-8) where {T}
+    # If |B G| ≤ Gmax, then
+    # |Gi| ≤ |e_i^T B^-1 B G| ≤ |B^-T e_i| Gmax = |A_i| Gmax
+    # with B the reciprocal lattice matrix and A_i the i-th column of the lattice matrix
+    Glims_ = [norm(lattice[:, i]) / 2T(π) * Gmax for i in 1:3]
```

Comment:
I don't like this variable name. It's too similar to `Glims` and easy to overlook the `_`. Change one or the other.